src/sentry/snuba/metrics/naming_layer/mri.py

```diff
@@ -157,6 +158,10 @@ class SpanMRI(Enum):
     HTTP_ERROR_RATE_LIGHT = "e:spans/http_error_rate_light@ratio"
 
 
+class ErrorsMRI(Enum):
```

Comment:
Isn't the name `ErrorsMRI` too broad? Are we planning to add error related metrics in the future?